code/modules/atmospherics/machinery/components/binary_devices/thermomachine.dm

```diff
@@ -220,7 +224,13 @@
 		investigate_log("had its internal [holding] swapped with [tank] by [key_name(user)].", INVESTIGATE_ATMOS)
 		replace_tank(user, tank)
 		update_appearance()
+		return
 
+	if(!anchored && item.tool_behaviour == TOOL_MULTITOOL)
```

Comment:
why !anchored?

Reply:
yeah i'll make it with only the panel open